src/submission/fields.py

```diff
@@ -0,0 +1,6 @@
+from django import forms
+
+
+class EditorSectionChoiceField(forms.ModelChoiceField):
```

Comment:
This class doesn't actually enforce any behaviour that warrants the "Editor" part of the name and since it is a such a niche use case, I wonder if it doesn't make more sense to just do this on `EditorArticleInfoForm.__init__()` instead, until the abstraction makes more sense

Reply:
I thought of doing this in `EditorArticleInfoForm.__init__()` as well but chose to do it this way to avoid repeating the logic around `FILTER_PUBLIC_FIELDS` that exists in the `ArticleInfo` form.